tests/functional/test_install_config.py

```diff
@@ -105,7 +105,10 @@ def test_command_line_append_flags(script, virtualenv, data):
         "Analyzing links from page https://test.pypi.org"
         in result.stdout
     )
-    assert "Skipping link %s" % data.find_links in result.stdout
+    assert (
+        'Skipping link: not a file: {}'.format(data.find_links) in
+        result.stdout
+    ), 'stdout: {}'.format(result.stdout)
```

Comment:
IIRC this can just be `, result`, right?

Reply:
I tried this out, and what happens is it adds other sections of info like `-- stderr: ---`, `-- created: ----`, and `-- updated: ---` (in addition to `-- stdout: ----`). Since those other sections can also be long and aren't needed, I think it might be better not to include them.

Reply:
I'd say they're useful diagnostic information (like an error getting logged on stderr by the networking stuff) or something like that.

I'll leave it to you to decide on what to include here since it's not worth too long a discussion anyway.

Reply:
I'll just leave it off. Normally, the test runner includes lots of surrounding info already. It's just when you can't see the string being checked against where it gets annoying (which was the reason for adding it).